python/cudf/cudf/tests/test_pandas_comparison.py

```diff
@@ -0,0 +1,366 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+import inspect
+import textwrap
+
+import numpy as np
+import pandas as pd
+import pytest
+
+import cudf
+from cudf.testing._utils import assert_eq
+
+
+def _is_cudf(lib):
+    # TODO: Is it safe to use `if lib is cudf`? I think there are some cases
+    # where the imports of a package in different modules may not be considered
+    # identical by `is` so we have to check the name as below, but this is
+    # probably worth verifying.
+    return lib.__name__ == "cudf"
+
+
+# The parameter name used for pandas/cudf in test functions.
+_LIB_PARAM_NAME = "lib"
+
+
+def pandas_comparison_test(*args, cudf_objects=None, assert_func=assert_eq):
```

Comment:
Is the suffix `_test` needed?
```suggestion
def pandas_comparison(*args, cudf_objects=None, assert_func=assert_eq):
```